backend/servers/express-webrouter/app/src/set.js

```diff
@@ -0,0 +1,17 @@
+import fs from 'fs'
+import path from 'path'
+
+const {
+ PORT,
+} = process.env
+
+export function setWithApp(app) {
+
+  app.set('port', (PORT || 5000))
+
+  const teleportDir = path.join(__dirname, '../../config/teleport.json')
```

Comment:
(because it is not the same thing actually, teleport_welcome is just a file for params that heps to build the welcome page )